Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/RenderViewportWidget.cpp

```diff
@@ -465,4 +465,14 @@ namespace AtomToolsFramework
     {
         return aznumeric_cast<float>(devicePixelRatioF());
     }
+
+    uint32_t RenderViewportWidget::GetMainDisplayRefreshRate() const
+    {
+        return 60;
+    }
+
+    uint32_t RenderViewportWidget::GetSyncInterval() const
+    {
+        return 1;
+    }
```

Comment:
Why are these default implemented? We should be able to retrieve the native window from the viewport context's window context (I added pass-through for DPI scaling for convenience, but we should still be able to just directly grab the window).

Reply:
These need to come from Qt